### PR TITLE
# CP-9 landing page fixes

### PR DESCRIPTION
**Pull request by samyshyaka**
#### What the PR does
Fix bugs of the landing page

#### Description of Tasks to be completed

- [x] Responsiveness is improved
- [x] Users should see different states for interactive elements
- [x] Skeleton UIs/screen are/is shown when the page loads/is loading
- [x] All tests are written

#### How this should be manually tested
Clone the project repository linked [here](https://github.com/atlp-rwanda/phantom-fe-codebandits)

```
cd phantom-fe-codebandits/
git checkout bg-landing-page-fixes-cp-9
git pull origin bg-landing-page-fixes-cp-9
npm install
npm run start
run http://localhost:3000 in your browser
```

#### Relevant Trello Card
[#CP-9](https://trello.com/c/X1Wfyki5/9-landing-page-fixes)

### CODE REVIEW

File: src/components/__test__/Button.test.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
 import React from 'react';
```
Comment (Jkadhuwa) [inline src/components/__test__/Button.test.js]: Rename the test folder to __tests__ for them to be ignored by codeclimate

File: src/components/LanguageDropdown.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ import React from 'react';
 
 const LanguageDropdown = ({ styles }) => {
   return (
-    <ul className=" z-10">
```
Comment (Jkadhuwa) [inline src/components/LanguageDropdown.js:5]: Use select tag in place of the list

**Review by Jkadhuwa — CHANGES_REQUESTED** [review]
Nice work on the tests, find some comments below, and also on mobile view increase the card to accommodate the languages when the button is clicked.

**Review by Jkadhuwa — DISMISSED** [review]

**Review by Yvad60 — COMMENTED** [review]
Great job man! the only thing I noticed is that the buttons on the header are a little bigger, it looks fine on the landing page but since the header will be used on other pages those buttons can appear to be bigger than the rest of page content 
also that the same with the skeleton UI they look bigger and too gray considering the background color of the page

**Review by Abdulkeza — COMMENTED** [review]
First of All, you did a great Job, Sam, so one thing, just if you take a look at Mobile Version (320px), the contents are breaking down, if you can , just you can add responsive for that

**Comment by samyshyaka** [comment]
> First of All, you did a great Job, Sam, so one thing, just if you take a look at Mobile Version (320px), the contents are breaking down, if you can , just you can add responsive for that

Thanks, Abdul,
I think 375px is the minimum of normal mobile devices

**Comment by samyshyaka** [comment]
> Great job man! the only thing I noticed is that the buttons on the header are a little bigger, it looks fine on the landing page but since the header will be used on other pages those buttons can appear to be bigger than the rest of page content also that the same with the skeleton UI they look bigger and too gray considering the background color of the page

Let me decrease them a little bit

**Review by Jkadhuwa — APPROVED** [review]

**Pull request merged**